README.md

```diff
@@ -1 +1,5 @@
-# cesium-webstorm-plugin
\ No newline at end of file
+# Cesium WebStorm Plugin
+
+[Download the plugin](https://plugins.jetbrains.com/plugin/8391?pr=).
```

Comment:
Include more instructions.  Assuming I don't know how to use Webstorm plugins, what exactly do I do?  Be concise.